completion_aggregator/tasks/aggregation_tasks.py

```diff
@@ -310,20 +310,22 @@ def resolve_stale_completions(self, changed_blocks, start):
 
 
 @shared_task
-def migrate_batch(offset, batch_size):  # Cannot pass a queryset to a task.
+def migrate_batch(start, stop):  # Cannot pass a queryset to a task.
     """
     Convert a batch of CourseModuleCompletions to BlockCompletions.
 
-    Given an offset and batch_size, this task will:
+    Given a starting ID and a stopping ID, this task will:
 
-    * Fetch a subset of the existing CourseModuleCompletions,
-    * Update the BlockCompletions table
+    * Fetch all CourseModuleCompletions with an ID in range(start_id, stop_id).
+    * Update the BlockCompletion table with those CourseModuleCompletion
+      records.
     """
     if not PROGRESS_IMPORTED:
         log.error("Cannot perform migration: CourseModuleCompletion not importable.")
 
-    queryset = CourseModuleCompletion.objects.all().order_by('id')
-    course_module_completions = queryset[offset:offset + batch_size]
+    log.error("Migrating batch from {} to {}".format(start, stop))
```

Comment:
Shouldn't this be ``log.info``?

Reply:
Yes, and it looks like we can actually delete that altogether, as it gets logged below on success.